Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -96,6 +55,8 @@ open class PassiveLocationManager: NSObject {
      */
     public weak var delegate: PassiveLocationManagerDelegate?
     
+    // MARK: Controlling Location Manager
```

Comment:
```suggestion
    // MARK: Starting and Stopping the Location Manager
```

Reply:
I tent to keep this topic a bit more abstract as there is also `updateLocation` method listed, which is not exactly 'stopping' the LocationManager.